src/app/screens/Home/index.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import dataBooks from "../../../data/books.json";
+import Home from "./layout.js"
+import "./styles.css";
+
+class HomeContainer extends Component {
+  state = { books: [], filterType:"Seleccionar",filter:""}
```

Comment:
declare `Seleccionar` and `Nombre`  as constants. And use them instead of hardcoding it

```
const select = 'Seleccionar';
const name = 'Nombre';

class HomeContainer .....
```